packages/braintree-utils/src/utils/is-credit-card-instrument-like.ts

```diff
@@ -0,0 +1,17 @@
+import { CreditCardInstrument } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+export default function isCreditCardInstrumentLike(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    instrument: any,
```

Comment:
Did you try to use `unknown` type? So it might help you to remove eslint disable rule

Reply:
fixed